src/app/auth/authentication.service.ts

```diff
@@ -61,7 +61,9 @@ export class AuthenticationService {
       let refreshInMs = Math.round(refreshInSeconds * .9) * 1000;
       console.log('Refreshing token in: ' + refreshInMs + ' milliseconds.');
       this.refreshInterval = refreshInMs;
-      this.clearTimeoutId = setTimeout(() => this.refreshToken(), refreshInMs);
+      if (process.env.ENV !== 'inmemory') {
```

Comment:
There is nothing of the planner nor the platform in this module. This change though would not rely on them, it is not exactly standard. What does this change affect?